pychromecast/socket_client.py

```diff
@@ -565,8 +565,14 @@ def run_once(self, timeout=POLL_TIME_NON_BLOCKING):
 
         # poll the socket, as well as the socketpair to allow us to be interrupted
         rlist = [self.socket, self.socketpair[0]]
+        # Map file descriptors to socket objects
```

Comment:
```suggestion
        # Map file descriptors to socket objects because select.select does not support fd > 1024
        # https://stackoverflow.com/questions/14250751/how-to-increase-filedescriptors-range-in-python-select
```